src/components/pages/Upload/UploadCase.js

```diff
@@ -222,44 +214,31 @@ const UploadCase = ({ authState }) => {
           Upload A Case
         </Button>
         <Modal
-          title="Upload Files"
+          title=""
           visible={isModalVisible}
           onOk={handleOk}
           onCancel={handleCancel}
+          footer={[
+            <Button key="back" onClick={handleCancel}>
+              Not Now
```

Comment:
I like that there's this option to return to the case! :)

Reply:
Agreed! Nice job incorporating that UX chnages.